lib/Support/ErrorHandling.cpp

```diff
@@ -85,7 +92,15 @@ static void klee_vfmessage(FILE *fp, const char *pfx, const char *msg,
                        /*bg=*/false);
   }
 
-  fdos << "KLEE: ";
+  if (LogCpuTime) {
+    sys::TimeValue now(0,0), user(0,0), sys(0,0);
+    sys::Process::GetTimeUsage(now,user,sys);
+    fdos << (user.seconds() + sys.seconds()) << ".";
+    fdos << (user.nanoseconds() + sys.nanoseconds());
```

Comment:
I think cpu time is more meaningful compared to elapsed time, when it comes to performance.